site/topics/particle-swarm-optimization/pso.rst

```diff
@@ -27,6 +27,47 @@ Particles
 Velocity
 ========
 
+* The velocity determines where the particle will be for the next iteration of the algorithm
+* In other words, the velocity :math:`\vec{v}(t)` is used to determine the position of particle :math:`\vec{x}(t+1)`
+
+
+Velocity Calculation
+--------------------
+
+* Velocities are typically initialized with some random values within some range
+* But as the algorithm executes, the velocity of the particles change as they become influenced by
+
+    * The particles' best known position in space
+    * The population's best known position in space
+
+
+* Velocity update for some particle :math:`i`
```

Comment:
I'd probably invert this. Build up the components then land at the linear combination.

Reply:
100% agree. 

I'm going to leave it as is here in this PR but then move it at the end of the next PR. with a subsection on like "tying it together"